qa/rspec/commands/base.rb

```diff
@@ -29,7 +29,7 @@ def run_command(cmd, host)
 
       response = nil
       at(hosts, {in: :serial}) do |_host|
-        response = sudo_exec!(cmd)
+        response = sudo_exec!("JARS_SKIP='true' #{cmd}")
```

Comment:
yes, a lot, see https://github.com/elastic/logstash/issues/5523 for more details.